_posts/2020-05-07-camel-kafka-connectors.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title:  "Using the Apache Camel Kafka Connectors with Strimzi"
+date: 2020-05-07
+author: sjwoodman
+---
+
+The Apache Camel project has just [released](https://camel.apache.org/blog/Camel-Kafka-connector-release-0.1.0/) a set of connectors which can be used to leverage the broad ecosystem of Camel in Kafka Connect. 
+Apache Camel is the leading Open Source integration framework enabling users to connect to applications which consume and produce data. 
+Camel enables a wide range of enterprise integration patterns and has connectors for over 300 different systems - from ActiveMQ to ZooKeeper.  
```

Comment:
do we want to add a link to this for example? https://camel.apache.org/components/latest/index.html

Reply:
Good idea.

Reply:
> LGTM. Just one comment and I was wondering why we are using "CustomResource" instead of "custom resource".

Perhaps incorrectly, I usually capitalise it that way. Looking at other posts I see "custom resource" is more common so have changed to that.

Reply:
Yeah, I think that when we are referring to a resource, so to a `KafkaConnector` resource in this case, then we use "custom resource". Instead, when referring to the concept of CRDs we use the capitalized version.